color_generator/datasets/dataloaders.py

```diff
@@ -0,0 +1,83 @@
+import numpy as np
+from torch.utils.data import DataLoader, Subset
+from torch.utils.data.sampler import SubsetRandomSampler
+
+
+class DataLoaders:
```

Comment:
Nicely done but it could be omitted by using datasets (https://huggingface.co/docs/datasets/quicktour.html#quick-tour)